dev/com.ibm.ws.jaxrs.2.0.client/resources/com/ibm/ws/jaxrs20/client/internal/resources/JAXRSClientMessages.nlsprops

```diff
@@ -48,3 +48,11 @@ failed_to_extract_saml_token_from_subject.useraction=Ensure that the server.xml
 failed_run_as_subject=CWWKW0706E: An exception occurred while attempting to get the RunAsSubject. The exception was: [{0}].
 failed_run_as_subject.explanation=An exception occurred while getting the RunAsSubject.
 failed_run_as_subject.useraction=Make sure the server is configured and started properly.
+
+warn_missing_mpjwt_token=CWWKW0707W: The [{0}] attribute in the [{1}] configuration is set to [{2}] but the MicroProfile JWT token is not available. The request will not contain an Authorization header with the token.
```

Comment:
- Add a comma after "[{2}]." It should be "...is set to [{2}], but the MicroProfile...."
- If the user could possibly misunderstand what "JWT" is, spell out what the acronym stands for, such as, "...JSON Web Token (JWT) is not available." Also, since "token" is part of the acronym, you do not need to say "JWT token." You could just say "JWT" instead. :)
- We try to use present tense as much as possible, so the second sentence should be "The request does not contain an Authorization...."

Reply:
Here's what the message would look like with the changes described above:
The [{0}] attribute in the [{1}] configuration is set to [{2}], but the MicroProfile JSON Web Token (JWT) is not available. The request does not contain an Authorization header with the token.